src/schema/core.yaml

```diff
@@ -126,6 +127,68 @@ enums:
       liquid:
       gas:
 
+  ChemicalEntityEnum:
+    permissible_values:
+      acetonitrile:
+        meaning: CHEBI:38472
+      alphaLP:
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of threonine, alanine, serine, and valine.
+      ammonium_acetate:
+        meaning: CHEBI:62947
+      ammonium_bicarbonate:
+        meaning: CHEBI:184335
+      Arg-C:
+        meaning: MS:1001303
+        description: >-
+          A cysteine protease that hydrolyzes peptide, ester, and amide bonds at the C-terminus of arginine and with 
+          lower efficiency, lysine.
+      Asp-N:
+        meaning: MS:1001304
+        description: >-
+          A zinc metalloendopeptidase that hydrolyzes peptide bonds at the N-terminus of aspartic acid.
+      chloroform:
+        meaning: CHEBI:35255
+      chymotrypsin:
+        meaning: MS:1001306
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of tryptophan, leucine, tyrosine, 
+          and phenylalanine.
+      formic_acid:
+        meaning: CHEBI:30751
+        description: >-
+          A carboxylic acid that is the simplest aliphatic carboxylic acid, comprising a hydrogen atom 
+          joined to the methyl group of methanol.
+      glucose:
+        meaning: CHEBI:17234
+        description: >-
+          Generally considered the most abundant monosaccharide in nature.
```

Comment:
descriptions should generally be drawn form the same source as the meaning and be definitional, not  commentary. ChEBI's definition for glucose is "An aldohexose used as a source of energy and metabolic intermediate."